Components/Badge/badge.css

```diff
@@ -0,0 +1,35 @@
+.badge-container{
+    position: relative;
+    width: fit-content;
+    display: flex;
+    margin: 2rem;
+}
+
+.bag-icon{
+    width: 2em;
+}
+
+.message-icon{
+    width: 2em;
+}
+
+.notification{
+    position: absolute;
+    background-color: red;
```

Comment:
Can you please declare gloabl variables for this color and use that at this place?